### PR TITLE
Master - Updating version

### PR DESCRIPTION
**Pull request by roigcarlo**
As we (I) always forget to update this. I will make the habit of doing this just after the release branch has been created.

The idea is to change dev by a number in the release branch in case we need. If you don't like I leave with 0 as before.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -19,9 +19,9 @@ if(POLICY CMP0074)
 endif(POLICY CMP0074)
 
 # Set here the version number **** only update upon tagging a release!
-set (KratosMultiphysics_MAJOR_VERSION 7)
-set (KratosMultiphysics_MINOR_VERSION 1)
-set (KratosMultiphysics_PATCH_VERSION 0)
+set (KratosMultiphysics_MAJOR_VERSION 9)
```
Comment (philbucher) [inline CMakeLists.txt:22]:
```suggestion
set (KratosMultiphysics_MAJOR_VERSION 8)
```

???

Comment (roigcarlo) [reply CMakeLists.txt:22]: No, release is 8, dev should be 9.

Comment (philbucher) [reply CMakeLists.txt:22]: ah ok

**Comment by philbucher** [comment]
doesn't compile

**Comment by roigcarlo** [comment]
I see, I will fix it when I have 5 min.

**Review by philbucher — APPROVED** [review]

**Pull request merged**